base/Dockerfile

```diff
@@ -12,10 +12,14 @@ RUN conda install --yes \
     nomkl \
     numpy==1.16.2 \
     pandas==0.24.2 \
-    && conda clean -tipsy
+    && conda update conda -y \
+    && conda clean -tipsy \
+    && find /opt/conda/ -type f,l -name '*.a' -delete \
+    && find /opt/conda/ -type f,l -name '*.pyc' -delete \
```

Comment:
These will probably be regenerated on first import. My guess is the actual files are smallish ~1KB. That said, would be curious to know if that matches with your experience or not. For instance how much space do all of the pyc files take up?